tests/bunit.generators.tests/bunit.generators.tests.csproj

```
@@ -16,6 +18,7 @@
 		<PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
 		<PackageReference Include="Verify.SourceGenerators" Version="2.2.0" />
 		<PackageReference Include="Verify.Xunit" Version="22.8.0" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="8.0.0" />
 		<PackageReference Update="Microsoft.CodeAnalysis.CSharp" Version="4.8.0" />
 		<PackageReference Update="Microsoft.CodeAnalysis.Common" Version="4.8.0" />
 		<PackageReference Update="Microsoft.CodeAnalysis.Workspaces.Common" Version="4.8.0" />
```

```suggestion
		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0" />
		<PackageReference Include="Microsoft.CodeAnalysis.Common" Version="4.8.0" />
		<PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="4.8.0" />
```

Cant remember of this shouldn't just be Inlucde instead of Update?